### DISCUSSION TITLE
login screen stays after login

### DISCUSSION THREAD

**Discussion by ThatSadCarGuy** [i3/i3#6185] [Q&A] [answered]
hi!
So I just installed i3 alongside KDE plasma on Endevouros (everything is updated) and i keep getting a problem, where when i choose the i3 session on the login screen and type in my password, i get the i3 bar at the bottom, but the background isn't black, it's my login screen. everything else works as it should, but that thing is kinda broken on my end. could someone help?

**Comment by okraits** [comment, marked as answer]
Is the desktop an active element (you can click it and menus pop up) or is it just a graphical issue?

If the latter is the case, then a workaround would be to add the following to your i3 config:

```
exec xsetroot -solid black
```

**Reply by ThatSadCarGuy** [reply]
Its just an graphical issue. And if i want to use a wallpaper later on, will I be able to? (I mean when i add that line)

**Reply by okraits** [reply]
Sure.

**Reply by ThatSadCarGuy** [reply]
Tried it, fixed now. Thanks :))
